src/paperqa/prompts.py

```diff
@@ -113,10 +113,12 @@
     " Respond with the following JSON format:"
     '\n\n{{\n  "summary": "...",\n  "relevance_score": "..."\n  "used_images"\n}}'
     "\n\nwhere `summary` is relevant information from the text - {summary_length} words."
-    " `relevance_score` is an integer 1-10 for the relevance of `summary` to the question."
+    " `relevance_score` is an integer 0-10 for the relevance of `summary` to the question."
```

Comment:
Should we add this `0-10` thing to the JSON snippet as well

```python
    '  "relevance_score": 0-10\n'  # putting numbers here, instead of "...", to get them as integer type not string
```

Reply:
Done in https://github.com/Future-House/paper-qa/pull/1107